tools/make/ffi.toml

```diff
@@ -97,18 +97,19 @@ description = "Build ICU4X CAPI for Cortex"
 category = "ICU4X FFI"
 dependencies = ["install-cortex-8-nightly"]
 toolchain = "${PINNED_CI_NIGHTLY}"
-env = { RUSTFLAGS = "-Ctarget-cpu=cortex-m33 -Cpanic=abort -Copt-level=s" }
+env = { RUSTFLAGS = "-Zunstable-options -Ctarget-cpu=cortex-m33 -Cpanic=immediate-abort -Copt-level=s" }
 command = "cargo"
 args = ["check", "--package", "icu_freertos",
         "--target", "thumbv8m.main-none-eabihf",
         "--no-default-features", "--features=wearos",
-        "-Zbuild-std=core,alloc", "-Zbuild-std-features=panic_immediate_abort"]
+        "-Zbuild-std=core,alloc"]
 
 [tasks.check-noalloc]
 description = "Build crates that are expected to work in no-alloc mode"
 category = "ICU4X FFI"
 dependencies = ["install-unknown-linux-nightly"]
 toolchain = "${PINNED_CI_NIGHTLY}"
+env = { CARGO_PROFILE_DEV_PANIC="abort" }
```

Comment:
what's `CARGO_PROFILE_DEV_PANIC`?

Reply:
Does RUSTFLAGS replace it or add to it?

Seems like we should either use this everywhere or at comment on what it is when we do use it. I point people to these files to explain how to do size-optimized builds; these files _are_ the documentation and we should treat them that way

Reply:
It adds to it.

This particular test is not the documentation for anything, and it's the only place we set a _single_ RUSTFLAG for panic=abort, so I used it here.

I undid the change and now we use RUSTFLAGS here instead. We should consider moving to CARGO_PROFILE_DEV_PANIC for our tests, potentially, but it's probably not worth it.